docs/code_review.md

```diff
@@ -35,3 +35,7 @@
     changes, and once all comments are addressed, you can squash related
     commits together and force push to the branch. Then we will just merge
     whole PR.
+* If the pull request introduces a workaround, reviewer should check that
+  rules noted in [Tracking of workarounds](./workarounds.md) page are
+  followed. Person merging a workaround is responsible of checking that all
+  information related to the workaround are correct.
```

Comment:
information ... is correct

Reply:
fixed in 33c39585412ebb9c08b22d4a732148151ea4ee0c